Google.PowerShell/Logging/GcLogCmdlet.cs

```diff
@@ -638,6 +638,61 @@ protected override void ProcessRecord()
 
     /// <summary>
     /// <para type="synopsis">
+    /// Gets StackDriver logs from a project.
+    /// </para>
+    /// <para type="description">
+    /// Gets StackDriver logs from a project. Will display logs from the default project if -Project is not used.
```

Comment:
It would probably be good to add some context about logs vs log entries and how to get them.